src/libraries/System.Private.CoreLib/src/System/DateTime.cs

```diff
@@ -980,6 +980,45 @@ internal void GetDatePart(out int year, out int month, out int day)
             day = n - days[m - 1] + 1;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Testing against the tight loop over 100,000 dates in System.Text.Json.Tests.Perf_DateTimes.WriteDateTimes, the total times worsen by about ~0.2ms, which translates to ~2ns per call. Because they the GetTime and GetTimePrecise methods are relatively small, and because there are currently only 8 usages, I think AggressiveInlining is the best option here.

Reply:
@jkotas, you ok with this attribution?

Reply:
I am ok with this one.

The improvement is not much (2ns / 60ns total), but the code expansion is not much either.